bot/cogs/snekbox.py

```diff
@@ -30,7 +30,8 @@
 try:
 {CODE}
 except Exception as e:
-    print("{}: {}".format(e.__class__.__name__, e))
+    import traceback
```

Comment:
This should be at the top of the file with the rest of the imports

Reply:
that wouldn't work, as discussed in #meta.